src/library_safetimers.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
I wonder if we can come up with a better name for this library?   I'm not really sure what "safe timers" means.  Do you know the history of this name or what it really means?

Reply:
I think they're "safe" in that the callbacks won't be run (immediately) if it's paused, or if the module has aborted.